aea/cli/utils/config.py

```diff
@@ -19,7 +19,7 @@
 # -*- coding: utf-8 -*-
 # ------------------------------------------------------------------------------
 #
-#   Copyright 2018-2020 Fetch.AI Limited
```

Comment:
Why does this file have two headers? Please delete if it's a mistake.

Reply:
done